pilot/pkg/security/authz/policy/v1alpha1/v1alpha1_test.go

```diff
@@ -209,6 +209,35 @@ func TestV1alpha1Generator_Generate(t *testing.T) {
 					policy.CustomPrincipal("td2", "binding", "binding")},
 			},
 		},
+		{
+			name: "one role and one binding with * in principal",
+			policies: []*model.Config{
+				policy.SimpleRole("role", namespaceA, serviceFoo),
+				policy.SimpleBinding("binding", namespaceA, "role", "*"),
+			},
+			trustDomain:        "td1",
+			trustDomainAliases: []string{"cluster.local", "td2"},
+			wantRules: map[string][]string{
+				"role": {policy.RoleTag("role"),
+					"*"},
```

Comment:
Please do not just put a raw string here, the test `policy.Verify()` is doing string search to find some specific pattern in the result, that's why we need those `policy.RoleTag` to make it specific, otherwise it could lead to false positive results in the test that the search matched with pattern that is not from your input.